src/main/java/com/google/sps/servlets/RepUploadImageServlet.java

```diff
@@ -74,7 +74,7 @@ private String getBlobKeyUrl(HttpServletRequest request, String formInputElement
         if (blobKeys == null || blobKeys.isEmpty()) {
             return(Constants.DEFAULT_POLITICIAN_IMAGE_URL);
         } else {
-            return (SERVE_URL + blobKeys.get(0).getKeyString());
```

Comment:
you want this space, you actually want it in the line above. I'd also move away from adding the parentheses unless its a really complex statement.